### DISCUSSION TITLE
Finding vulnerabilities for a specific host

### DISCUSSION THREAD

**Discussion by secAnalyst** [CrowdStrike/psfalcon#199] [Q&A]
Is it possible to filter Get-FalconVulnerability to list all the vulnerabilities on a given host? If not, is there another command that will allow this? I have tried Get-FalconVulnerability -Filter "host_info.hostname:'<host>'" and it doesn't seem to be working. Thanks for any help!

**Comment by bk-cs** [comment]
Hostname is not an allowed filter, as far as I'm aware. The documentation specifies `aid`, which is the specific device's identifier value:

```
Get-FalconVulnerability -Filter "aid:'device_id_value'" [-Detailed] [-All]
```

\[ [US-1](https://falcon.crowdstrike.com/documentation/98/spotlight-apis#spotlight-api-filter-parameters) | [US-2](https://falcon.us-2.crowdstrike.com/documentation/98/spotlight-apis#spotlight-api-filter-parameters) | [EU-1](https://falcon.eu-1.crowdstrike.com/documentation/98/spotlight-apis#spotlight-api-filter-parameters) | [US-GOV-1](https://falcon.laggar.gcw.crowdstrike.com/documentation/98/spotlight-apis#spotlight-api-filter-parameters) \]
